.travis.yml

```diff
@@ -18,16 +19,22 @@ gemfile:
   - gemfiles/rails_4_2.gemfile
 matrix:
   exclude:
-    - rvm: 2.6.5
+    - rvm: 2.7.1
       gemfile: gemfiles/rails_4_2.gemfile
-    - rvm: 2.4.9
+    - rvm: 2.6.6
+      gemfile: gemfiles/rails_4_2.gemfile
+    - rvm: 2.4.10
       gemfile: gemfiles/rails_6_0.gemfile
-cache: bundler
+cache:
+  directories:
+    - vendor/bundle
 # Source: <https://docs.travis-ci.com/user/languages/ruby/#bundler-20>
 before_install:
-  - gem update --system '2.7.8' --no-document
+  - rm -f ~/.rvm/rubies/ruby-*/bin/bundle
+  - gem update --system --force --no-document
```

Comment:
This is necessary because I was getting deadlock issues when staying on a version of RubyGems prior to 3.1.0.